controllers/help_request.js

```diff
@@ -5,6 +5,45 @@ var UserModel = require("../models/user").model;
 var NotificationService = require("../services/notification.service");
 var UserService = require("../services/user.service");
 
+const putHelpRequest = async (req, res) => {
+  let status = req.body.status;
+  let newResponder = req.body.newResponder;
```

Comment:
if this is not supplied, the function will push {} to the array of responderIds of the help request

Reply:
fixed